usaddress/__init__.py

```diff
@@ -61,72 +61,81 @@
+    'isle', 'isles', 'islnd', 'islnds', 'iss', 'jct', 'jction', 'jctn',
+    'jctns', 'jcts', 'junction', 'junctions', 'junctn', 'juncton', 'key',
+    'keys', 'knl', 'knls', 'knol', 'knoll', 'knolls', 'ky', 'kys', 'la',
+    'lake', 'lakes', 'land', 'landing', 'lane', 'lanes', 'lck', 'lcks', 'ldg',
+    'ldge', 'lf', 'lgt', 'lgts', 'light', 'lights', 'lk', 'lks', 'ln', 'lndg',
+    'lndng', 'loaf', 'lock', 'locks', 'lodg', 'lodge', 'loop', 'loops', 'lp',
+    'mall', 'manor', 'manors', 'mdw', 'mdws', 'meadow', 'meadows', 'medows',
+    'mews', 'mi', 'mile', 'mill', 'mills', 'mission', 'missn', 'ml', 'mls',
+    'mn', 'mnr', 'mnrs', 'mnt', 'mntain', 'mntn', 'mntns', 'motorway',
+    'mount', 'mountain', 'mountains', 'mountin', 'msn', 'mssn', 'mt', 'mtin',
+    'mtn', 'mtns', 'mtwy', 'nck', 'neck', 'opas', 'orch', 'orchard', 'orchrd',
+    'oval', 'overlook', 'overpass', 'ovl', 'ovlk', 'park', 'parks', 'parkway',
+    'parkways', 'parkwy', 'pass', 'passage', 'path', 'paths', 'pike', 'pikes',
+    'pine', 'pines', 'pk', 'pkway', 'pkwy', 'pkwys', 'pky', 'pl', 'place',
+    'plain', 'plaines', 'plains', 'plaza', 'pln', 'plns', 'plz', 'plza',
+    'pne', 'pnes', 'point', 'points', 'port', 'ports', 'pr', 'prairie',
+    'prarie', 'prk', 'prr', 'prt', 'prts', 'psge', 'pt', 'pts', 'pw', 'pwy',
+    'rad', 'radial', 'radiel', 'radl', 'ramp', 'ranch', 'ranches', 'rapid',
+    'rapids', 'rd', 'rdg', 'rdge', 'rdgs', 'rds', 'rest', 'ri', 'ridge',
+    'ridges', 'rise', 'riv', 'river', 'rivr', 'rn', 'rnch', 'rnchs', 'road',
+    'roads', 'route', 'row', 'rpd', 'rpds', 'rst', 'rte', 'rue', 'run', 'rvr',
+    'shl', 'shls', 'shoal', 'shoals', 'shoar', 'shoars', 'shore', 'shores',
+    'shr', 'shrs', 'skwy', 'skyway', 'smt', 'spg', 'spgs', 'spng', 'spngs',
+    'spring', 'springs', 'sprng', 'sprngs', 'spur', 'spurs', 'sq', 'sqr',
+    'sqre', 'sqrs', 'sqs', 'squ', 'square', 'squares', 'st', 'sta', 'station',
+    'statn', 'stn', 'str', 'stra', 'strav', 'strave', 'straven', 'stravenue',
+    'stravn', 'stream', 'street', 'streets', 'streme', 'strm', 'strt',
+    'strvn', 'strvnue', 'sts', 'sumit', 'sumitt', 'summit', 'te', 'ter',
+    'terr', 'terrace', 'throughway', 'tl', 'tpk', 'tpke', 'tr', 'trace',
+    'traces', 'track', 'tracks', 'trafficway', 'trail', 'trailer', 'trails',
+    'trak', 'trce', 'trfy', 'trk', 'trks', 'trl', 'trlr', 'trlrs', 'trls',
+    'trnpk', 'trpk', 'trwy', 'tunel', 'tunl', 'tunls', 'tunnel', 'tunnels',
+    'tunnl', 'turn', 'turnpike', 'turnpk', 'un', 'underpass', 'union',
+    'unions', 'uns', 'upas', 'valley', 'valleys', 'vally', 'vdct', 'via',
+    'viadct', 'viaduct', 'view', 'views', 'vill', 'villag', 'village',
+    'villages', 'ville', 'villg', 'villiage', 'vis', 'vist', 'vista', 'vl',
+    'vlg', 'vlgs', 'vlly', 'vly', 'vlys', 'vst', 'vsta', 'vw', 'vws', 'walk',
+    'walks', 'wall', 'way', 'ways', 'well', 'wells', 'wl', 'wls', 'wy', 'xc',
+    'xg', 'xing', 'xrd', 'xrds' 
```

Comment:
flake8 flagged a trailing whitespace here, which is part of why [the Travis build failed.](https://travis-ci.org/datamade/usaddress/jobs/247236325)